### ISSUE TITLE
[BUG] GUI had problems reporting "AttributeError: 'MainW' object has no attribute 'stack_filtered'"

### ISSUE THREAD

**Issue by xuanjiang102** [MouseLand/cellpose#882]
**Describe the bug**
The GUI is not working for running segmentation. After loading the gay scale image, the "run cyto3" button is gay and cannot be clicked. It was working fine with the same image the day before. There is an error message in the terminal whenever loading the image:

```
Traceback (most recent call last):
  File "/Users/username/opt/miniconda3/envs/cellpose/lib/python3.8/site-packages/cellpose/gui/gui.py", line 1142, in dropEvent
    io._load_image(self, filename=files[0], load_seg=True, load_3D=self.load_3D)
  File "/Users/username/opt/miniconda3/envs/cellpose/lib/python3.8/site-packages/cellpose/gui/io.py", line 114, in _load_image
    _load_seg(parent, manual_file, image=imread(filename), image_file=filename,
  File "/Users/username/opt/miniconda3/envs/cellpose/lib/python3.8/site-packages/cellpose/gui/io.py", line 348, in _load_seg
    print(parent.stack.shape, parent.stack_filtered.shape)
AttributeError: 'MainW' object has no attribute 'stack_filtered'
```

**To Reproduce**
1. start the GUI in terminal by running "python -m cellpose"
2. Drag the file into the GUI

**Run log**
Please post all command line/notebook output for us to understand the problem. For this please make sure you are running with verbose mode on. So command line, with `--verbose` tag, or in a notebook first run 
```
2024-03-04 16:48:23,560 [INFO] WRITING LOG OUTPUT TO /Users/username/.cellpose/run.log
2024-03-04 16:48:23,560 [INFO]
cellpose version: 	3.0.5
platform:       	darwin
python version: 	3.8.18
torch version:  	2.2.1
2024-03-04 16:48:24,012 [INFO] TORCH CUDA version not installed/working.
GUI_INFO: normalization checked: computing saturation levels (and optionally filtered image)
{'lowhigh': None, 'percentile': [1.0, 99.0], 'normalize': True, 'norm3D': False, 'sharpen_radius': 0, 'smooth_radius': 0, 'tile_norm_blocksize': 0, 'tile_norm_smooth3D': 1, 'invert': False}
[0, 255.0]
Traceback (most recent call last):
  File "/Users/username/opt/miniconda3/envs/cellpose/lib/python3.8/site-packages/cellpose/gui/gui.py", line 1142, in dropEvent
    io._load_image(self, filename=files[0], load_seg=True, load_3D=self.load_3D)
  File "/Users/username/opt/miniconda3/envs/cellpose/lib/python3.8/site-packages/cellpose/gui/io.py", line 114, in _load_image
    _load_seg(parent, manual_file, image=imread(filename), image_file=filename,
  File "/Users/username/opt/miniconda3/envs/cellpose/lib/python3.8/site-packages/cellpose/gui/io.py", line 348, in _load_seg
    print(parent.stack.shape, parent.stack_filtered.shape)
AttributeError: 'MainW' object has no attribute 'stack_filtered'
``` 

**Screenshots**

![error](https://github.com/MouseLand/cellpose/assets/79315927/a0321760-3f8e-4278-b786-19aa750cc217)

**Comment by carsen-stringer** [comment]
sorry about this, it's fixed now (#879), please `pip install git+https://github.com/mouseland/cellpose.git`

**Comment by xuanjiang102** [comment]
> sorry about this, it's fixed now (#879), please `pip install git+https://github.com/mouseland/cellpose.git`

Thanks, that solves the issue.
